### PR TITLE
Issue #552: Fix user_update schema version for Manage user accounts m…

### LINKED ISSUE #552: [UX] "Find user accounts" / "Administer users" should be "Manage user accounts"
...or alternatively "Manage users" or "Manage people".

We need to change the view label (currently "Administer users"), the menu bar link (currently "Find user accounts") because that page also allows actions to be made on listed accounts, so "manage" is a more fitting word.

Perhaps the page title too for consistency (currently "People").

### PR DESCRIPTION
**Pull request by serundeputy**
This fixes: https://github.com/backdrop/backdrop-issues/issues/552

### CODE REVIEW

File: core/modules/user/user.install [hunk 1]
```diff
@@ -1124,5 +1124,18 @@ function user_update_1011() {
...
+function user_update_1012() {
+  $config = config('views.view.user_admin');
+	if ($config->get('display.default.display_options.title') == 'User accounts') {
+    $config->set('display.default.display_options.title', 'Manage user accounts');
+  }
+  if ($config->get('display.page.display_options.menu.title') == 'Find user accounts') {
+    $config->set('display.page.display_options.menu.title', 'Manage user accounts');
+  }
+  $config->save();
+}
```
Comment (klonos) [inline core/modules/user/user.install:1138]: Nit-picking here, but should have a blank line there after the closing bracket.

**Comment by klonos** [comment]
Worked! Didn't even have to clear caches or anything. Thanx @serundeputy

File: core/modules/user/user.install (hunk 1 above)
Comment (jenlampton) [inline core/modules/user/user.install:1138]: :+1:

**Comment by quicksketch** [comment]
Great, thanks @serundeputy!

**Pull request merged**